src/test/kotlin/com/exactpro/th2/common/schema/grpc/router/impl/DefaultGrpcRouterTest.kt

```diff
@@ -289,10 +289,6 @@ internal class DefaultGrpcRouterTest {
             )
         }
 
-        /* FIXME: this test sometimes failing with TimeoutException
```

Comment:
Is the test fixed after merging with dev-release-5? Just checking if it was not removed accidentally

Reply:
I can't reproduce this problem now.

Reply:
Understood, thank you